handlers_global.go

```diff
@@ -185,11 +185,12 @@ func unmarshalMetricsFromHTTP(ctx context.Context, client *trace.Client, w http.
 	}
 
 	w.WriteHeader(http.StatusAccepted)
-	span.Add(ssf.Timing("import.response_duration_ns", time.Since(span.Start),
-		time.Nanosecond, map[string]string{
-			"part":     "request",
-			"encoding": encoding,
-		}))
+	span.Add(ssf.RandomlySample(0.1,
```

Comment:
I disagree, some reasons:
* configuring these things is difficult (you have to thread values across multiple layers of abstraction, and might even have to expose multiple knobs, and more importantly:
* making these configuration values is pretty much us just declaring "idk ¯\_(ツ)_/¯" when really, we should know the proportion of writes required to get useful information out of the system - setting these to too high will *always* result in write near-catastrophic amplification (:

I think we should expose a config value for *every other* metric though - the ones that get emitted with sample rate 1, instead; but that would require a bit of restructuring of the code (since they get sampled at batch Add time, not creation or submit).

Reply:
Yeah, i get the difficulty of threading config through, for sure. As well as the problem of simply exposing too many dials.

i did not realize that it is possible for us to make a generalized determination about what sampling thresholds here will still produce adequate information. i'd like to understand that better, in this context; this _seemed_ to me like a rather generic ingestion point, about which we can't necessarily reason that well in a general case.